src/OpenStudy/study_openinterface.jl

```diff
@@ -28,8 +30,19 @@ Base.@kwdef mutable struct Data{T}
     raw::T
     stage_type::StageType
 
+    data_path::String
+
+    rng::Random.MersenneTwister = Random.MersenneTwister(0)
```

Comment:
Why do you need a rng here?

Reply:
I was using for generating temp files, but not anymore. will remove